apiary.apib

```diff
@@ -541,6 +541,80 @@ API endpoints that Operation Code's Rails backend makes available to its React f
                 status: "ok"
             }
 
++ Response 422 (application/json)
+
+        {
+            errors: "Some error message"
+        }
+
+## Mentor | Collection [/api/v1/mentors/{?id,email,zip,first_name,last_name,slack_name,mentor,verified}]
++ Parameters
+
+    + id (integer, required) - Identification of mentor in the form of an integer
+    + email (string, required) - Email of the Mentor in the form of a string
+    + zip (string, optional) - String of comma-separated zip code(s), i.e. '80126', or '80126, 80203'
+    + first_name (string, required) - First Name of the mentor in the form of a string
+    + last_name (string, required) - Last Name of the mentor in the form of a string
+    + slack_name (string, required) - Slack Name of the mentor in the form of a string
+    + mentor (boolean, required) - Has the mentor been verified in the form of a boolean
+    + verified (boolean, required) - Has the mentor not been verified in the form of a boolean
+
+### List All Mentors [GET]
+
++ Response 200 (application/json)
+
+    + Body
+
+            {
+                "id":38,
+                "email":"jayden.gerhold@aufderhar.io",
+                "zip":"97201",
+                "first_name":"Loren",
+                "last_name":"Olson",
+                "slack_name":"Darth Caedus",
+                "mentor":true,
+                "verified":false
+            },
+
+
++ Response 422 (application/json)
+
+        {
+            errors: "Some error message"
+        }
+ 
+## Mentor | Show [/api/v1/requests/{?id,email,zip,first_name,last_name,slack_name,mentor,verified,bio}]
```

Comment:
The only parameter a `show` action will take is the `id` of the record to be shown.  

Here is an example for reference:  https://operationcodeapi.docs.apiary.io/#reference/0/resource-show-or-delete/view-a-resource's-detail